src/foam/u2/tag/TextArea.js

```diff
@@ -97,6 +97,19 @@ foam.CLASS({
       // TODO: make sure that DOM is updated if values don't change
       this.setAttribute('readonly', mode === foam.u2.DisplayMode.RO);
       this.setAttribute('disabled', mode === foam.u2.DisplayMode.DISABLED);
-    }
+    },
+
+    function fromProperty(p) {
+      this.SUPER(p);
+
+      if ( ! this.hasOwnProperty('onKey') ) {
+        if ( p.hasOwnProperty('onKey') ) this.onKey = p.onKey;
```

Comment:
Please indent this if/else "properly" ;-).

Reply:
Isn't that how you would indent one-line if-else statements? Either way `? :` looks better for this purpose so using that